packages/cmrjs/cmr-utils.js

```diff
@@ -187,22 +239,42 @@ function constructOnlineAccessUrls(files, distEndpoint, buckets) {
   files.forEach((file) => {
     const urlObj = {};
     const bucketType = buckets.type(file.bucket);
-
     if (bucketType === 'protected') {
       const extension = urljoin(file.bucket, `${file.filepath}`);
       urlObj.URL = urljoin(distEndpoint, extension);
       urlObj.URLDescription = 'File to download';
+      urlObj.Type = 'GET DATA';
       urls.push(urlObj);
     }
     else if (bucketType === 'public') {
       urlObj.URL = `https://${file.bucket}.s3.amazonaws.com/${file.filepath}`;
       urlObj.URLDescription = 'File to download';
+      urlObj.Type = 'GET DATA';
       urls.push(urlObj);
     }
   });
+
   return urls;
 }
 
+/**
+ * create a list of URL objects that should not appear in under online access in the CMR metadata.
```

Comment:
Nit: I think there is an extra preposition here
```
create a list of URL objects that should not appear in online access in the CMR metadata.
```
or 
```
create a list of URL objects that should not appear under online access in the CMR metadata.
```

Reply:
4bd7cb793